.github/workflows/test_and_deploy.yml

```diff
@@ -7,6 +7,7 @@ on:
   push:
     branches:
       - main
+      - cy/utils
```

Comment:
Because this PR targets another branch that is not main, the "on PR" condition is not fulfilled in Actions. So I'm overriding manually, this will be removed when I start merging all the PRs (which depend on each other)